front/pages/w/[wId]/welcome.tsx

```diff
@@ -93,22 +105,13 @@ export default function Welcome({
       body: JSON.stringify({ firstName, lastName }),
     });
     if (updateUserFullNameRes.ok) {
-      await fetch("/api/user/metadata/expertise", {
+      await fetch("/api/user/metadata/team", {
```

Comment:
Is team really the right name here?

Maybe more job type? + we can update the wording around that?